### PR TITLE
Target wyrihaximus/react-guzzle-ring 2.0 and 1.0

### PR DESCRIPTION
**Pull request by WyriHaximus**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,10 +12,17 @@
...
         "irc": "irc://irc.freenode.net/phergie"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/WyriHaximus-labs/phergie-irc-client-react"
+        }
+    ],
     "require": {
+        "phergie/phergie-irc-client-react": "dev-reactphp-1.0 as 3.1.0",
```
Comment (WyriHaximus) [inline composer.json]: This and the `repositories` section above are require until https://github.com/phergie/phergie-irc-client-react/pull/65 has been finished, merged, and tagged.

**Review by matthewtrask — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -30,5 +30,11 @@
         "psr-4": {
             "Phergie\\Tests\\Plugin\\Http\\": "tests"
         }
+    },
+    "config": {
+        "sort-packages": true,
+        "platform": {
+          "php": "7.6.19"
```
Comment (elazar) [inline composer.json]: Is this version number a typo?

Comment (matthewtrask) [reply composer.json]: oh snap, my eyes were crossed and missed this, good catch @elazar

**Comment by elazar** [comment]
5.6 build job is failing, seemingly because doctrine/instantiator requires PHP >= 7.1.

File: composer.json (hunk 2 above)
Comment (WyriHaximus) [reply composer.json]: No, just a random valid one above the lower versions to ensure we don't run into odd packages with odd minimum version requirements

**Comment by WyriHaximus** [comment]
> 5.6 build job is failing, seemingly because doctrine/instantiator requires PHP >= 7.1.

Correct :(. Looking into that now why it it still tries to install something it shouldn't :/

**Comment by WyriHaximus** [comment]
Might have a clue why it failed, testing that now

**Comment by WyriHaximus** [comment]
Ok fixed the build

**Review by elazar — APPROVED** [review]

**Pull request merged**